src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/MockHttpClientTransport.cs

```diff
@@ -23,7 +24,15 @@ public AddMockTransport(ConfigurationSetting responseContent)
             _expectedUri = $"https://contoso.azconfig.io/kv/{responseContent.Key}{GetExtraUriParameters(responseContent)}";
             _expectedMethod = HttpMethod.Put;
             _expectedRequestContent = GenerateExpectedRequestContent(responseContent);
-            _responseContent = CreateResponse(responseContent);
+            if(responseContent != null)
+            {
+                ReadOnlyMemory<byte> content = Serialize(responseContent);
+                _responseContent = Encoding.UTF8.GetString(content.Span.ToArray()).ToLowerInvariant();
+            }
+            else
+            {
+                _responseContent = null;
```

Comment:
What's the default value of _responseContent?

Reply:
hmm good point. It should be null...